### PR TITLE
Feat/schedule-component

### PR DESCRIPTION
**Pull request by andrereis26**
https://app.clickup.com/t/86bxh4bh7

This prolly gonna need some feedback before being approved! (like the colors or smth)

image

image

### CODE REVIEW

File: src/components/schedule/ScheduleContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import schedule from "../../data/schedule.json";
+
+const ScheduleContainer: React.FC = () => {
+    return (
+        <div className="w-full rounded-lg dark:bg-[#3b3b3b] bg-secondary flex">
```
Comment (andrefbarros) [inline src/components/schedule/ScheduleContainer.tsx]: Probably moving these colors to tailwing config wouldn't be a bad ideia

File: src/components/schedule/ScheduleContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            {schedule.map((day, index) => (
+                <div key={index} className={`flex-1 pl-8 py-8 ${index !== schedule.length - 1 ? 'border-r border-white' : ''}`}>
+                    <div className="flex items-center flex-1 pb-4">
+                        <div className="bg-neutral-400 dark:bg-[#000614] p-1 flex">
```
Comment (andrefbarros) [inline src/components/schedule/ScheduleContainer.tsx]: Same as here

File: src/components/schedule/Schedule.astro
Comment (andrefbarros) [inline src/components/schedule/Schedule.astro:1]: We need to make these columns stack vertically on mobile. It's possible that using the same approach to add a white line between days would also work. What do you think?

File: src/components/schedule/ScheduleContainer.tsx
Comment (andrefbarros) [inline src/components/schedule/ScheduleContainer.tsx:1]: In terms of the white theme, I do think we should opt for a darker gray for the text representing the days of the week, and for the background of the "card", perhaps a very pale gray or off-white. What do you think?

File: src/components/schedule/Schedule.astro
Comment (andrereis26) [reply src/components/schedule/Schedule.astro:1]: SGTM! Ngl but I totally forgot about mobile :p

File: src/components/schedule/ScheduleContainer.tsx
Comment (andrereis26) [reply src/components/schedule/ScheduleContainer.tsx:1]: LGTM, I didn't know what colors to choose tbh

Comment (andrereis26) [reply src/components/schedule/ScheduleContainer.tsx:1]: **Original (before)**
image

**Option 1** (text has a dark gray color)
image

**Option 2** (version committed)
image

Comment (andrefbarros) [reply src/components/schedule/ScheduleContainer.tsx:1]: Actually, I think the first version is better, the original, :smile: 
Another thing though, we still run into styling issues when the size of the screen is medium
image

**Review by andrefbarros — APPROVED** [review]

**Comment by andrefbarros** [comment]
Do not merge for now because we still need to know how the release will happen 😃

**Pull request merged**